Mugs/Mug Installer.event

```diff
@@ -118,3 +118,8 @@ setMugEntry(0x1C, Mariya_Mug, 4, 5, 4, 3)
 MichealP_Mug:
 #incext PortraitFormatter "MichealPMug.png"
 setMugEntry(0x26, MichealP_Mug, 4, 5, 4, 3)
+
+#define VergeinPortrait 0x74
```

Comment:
Looks like something that could go in `Custom Definitions.txt` (and done more).

Reply:
Yeah, in the ch1 branch I refactor mug defs a little bit. I do want to keep it in the mug installer, though.